modules/reference/pages/data-transform-rust-sdk.adoc

```diff
@@ -191,8 +191,15 @@ impl<'a> RecordWriter<'a> {
     // Creates a new [`RecordWriter`] using the specified `sink`.
     pub fn new(sink: &'a mut dyn RecordSink) -> Self;
 
-    /// Write a record to the output topic returning any errors.
+    /// Write a record to the default output topic, returning any errors.
```

Comment:
was this the one that they provided which topic it was?

Reply:
Not sure, @rockwotj can you comment?

Reply:
The "default" output topic is the 0th output topic in their config file or in their cli deploy `rpk transform deploy --output-topic foo --output-topic bar` The default topic in this case is `foo`.